app/stores/WalletsStore.js

```diff
@@ -262,4 +266,19 @@ export default class WalletsStore extends Store {
     });
   };
 
+  @action _hideWalletAddressCopyNotification = () => {
+    this.isWalletAddressCopyNotificationVisible = false;
+  }
+
+  _onShowWalletAddressCopyNotification = action(() => {
+    if (this._hideWalletAddressCopyNotificationTimeout) {
+      clearTimeout(this._hideWalletAddressCopyNotificationTimeout);
+    }
+    this._hideWalletAddressCopyNotificationTimeout = setTimeout(
+      this._hideWalletAddressCopyNotification,
+      10000
```

Comment:
Please move this magic constant for hiding delay to a config value. I see that we don't have anything like that yet … so let's create `app/config.js` and add 
```js
export default {
  wallets: {
    ADDRESS_COPY_NOTIFICATION_DURATION: 10000,
  }
};
```